projects/RR360/hooks/fps_hook.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+from mmengine.registry import HOOKS
+from mmengine.hooks import Hook
+from mmengine.device import get_max_cuda_memory, is_cuda_available
+
+import pymongo
```

Comment:
Tell users  in README.md that he needs to install `pymongo`.

Reply:
delete the benchmark so that won't use pymongo